client/PotLuck.css

```diff
@@ -21,41 +21,19 @@ label, input {
     padding:10px 0;
 }
 
-.potluck-app{
-    min-width:200px;
-    position:absolute;
-    left:0;
-    right:0;
-    top:0;
-    bottom:0;
-    background-color:rgb(200,150,150);
-}
-.potluck-app-header{
-    color:rgba(0,0,0,0.5);
-    padding:10px;
-    font-size:24px;
-    height:50px;
-    left:0;
-    right:0;
-    position:relative;
-}
-.potluck-app-header > *{
-    display:table-cell;
-}
-
 #NewPotLuckBtn{
     position:absolute;
-    right:10px;
-    top:10px;
+    left:650px;
```

Comment:
Because I could! :-D j/k   Thanks for pointing that out. I'm sure that I was just ignoring the code that was already there. I was distracted by the nonexistent cats.